tests/baselines/reference/templateStringsArrayTypeRedefinedInES6Mode.errors.txt

```diff
@@ -1,10 +1,8 @@
 lib.d.ts(521,11): error TS2300: Duplicate identifier 'TemplateStringsArray'.
 tests/cases/compiler/templateStringsArrayTypeRedefinedInES6Mode.ts(2,7): error TS2300: Duplicate identifier 'TemplateStringsArray'.
-tests/cases/compiler/templateStringsArrayTypeRedefinedInES6Mode.ts(8,3): error TS2345: Argument of type '{ [x: number]: undefined; }' is not assignable to parameter of type 'TemplateStringsArray'.

```

Comment:
Now `TemplateStringsArray` is resolved as the class. Not sure if we should add `@noLib` here to